rosdep/base.yaml

```diff
@@ -830,6 +830,8 @@ fcgi:
   fedora: [fcgi, mod_fcgid, spawn-fcgi]
   gentoo: [dev-ruby/fcgi]
   ubuntu: [libfcgi-dev, libapache2-mod-fastcgi, spawn-fcgi]
+fcl:
+  ubuntu: [libfcl0.5]
```

Comment:
Should the contents here perhaps just be a copy/paste of the `libfcl-dev` entry?